Source/Other/CategoryAdditions/SPNSMutableDictionaryAdditions.h

```diff
@@ -13,9 +13,9 @@ NS_ASSUME_NONNULL_BEGIN
 @interface NSMutableDictionary (SPNSMutableDictionaryAdditions)
 
 
-- (id)SPsafeObjectForKey:(id)key;
-- (void)SPsafeSetObject:(id)obj forKey:(id)key;
-- (void)SPsafeRremoveObjectForKey:(nullable id)key;
+- (id)safeObjectForKey:(id)key;
```

Comment:
Just a check (not sure) - isn't `key` in Foundation method String, not id?

Reply:
it's `(KeyType <NSCopying>)` ... I couldn't figure out how to use it...